spec/helpers/application_notifications_helper_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationNotificationsHelper, type: :helper do
+  let(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    describe '.notification_view_path' do
+      let!(:course) { create(:course) }
+      let!(:user) { create(:user, name: 'tester') }
+      let!(:activity) do
+        create(:activity, actor: user, object: course, event: :tested)
+      end
+      let!(:user_notification) do
+        create(:user_notification, user: user, activity: activity, notification_type: :popup)
+      end
+      let!(:course_notification) do
+        create(:course_notification, course: course, activity: activity, notification_type: :feed)
+      end
+
+      context 'when argument is a notification' do
```

Comment:
actually you should have individual tests for course notification and user notification.